plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PartitionFields.java

```diff
@@ -156,7 +149,7 @@ private static String fromColumnToIdentifier(String column)
 
     public static String quotedName(String name)
     {
```

Comment:
this also changes the sorted_by columns right? do we already test it?

Reply:
TestSortFieldUtils already has a test of uppercase and mixed-case. Added a few assertions just in case.